vue-custom-bootstrap/src/App.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div id="app">
+    <Assessment />
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+import BootstrapVue from "bootstrap-vue";
+
+import Assessment from "./views/Assessment.vue";
+
+export default Vue.extend({
+  name: "app",
+  components: { Assessment }
+});
+Vue.use(BootstrapVue);
+</script>
+
+<style lang="scss">
+#app {
+  margin: 0 auto;
+  max-width: 52rem;
+}
+
```

Comment:
Here is where we can set bootstrap theme variables. Right now I'm just changing the primary color to the City of Toronto color, but the full list of variables is here: https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss

Reply:
Excellent Russell.
One thing mightily should be considered,
"type" of each group is better from the JSON file.
As our goal is any changes in the backend , it should not need changing in the front end.

Thanks 

Qing

Reply:
<style lang="scss">
#app {
  margin: 0 auto;
  max-width: 52rem;
}
Is it possible to link style file from index.html?

qing